docs/api/mount/setMethods.md

```diff
@@ -0,0 +1,32 @@
+# setMethods(methods)
+
+Sets Vue instance methods and forces update. Can only be called on a Vue component wrapper.
+
+### Arguments
+
+props (`Object`): Methods to set.
+
+### Example
+
+`setMethods` is particularly useful for mocking or stubbing a method, for example mocking a method that makes an ajax request to an external resource.
```

Comment:
Thanks! I think the reason a lot of devs don't unit test is because of a lack of throrough examples. I am looking to change that (: